### DISCUSSION TITLE
need to quickly find the file that is being edited in Navigator/Project from SQL editor

### DISCUSSION THREAD

**Discussion by yyx100** [dbeaver/dbeaver#13422] [Q&A]
**Is your feature request related to a problem? Please describe.**
From the SQL Editor, the "Show In" (Alt+shift+w) feature is missing comparing to Text/Java Editor in Eclipse.  It is important because we often need to find the right click on the file in the Project or Navigator  panel to perform tasks like "team", "copy" ...

**Describe the solution you'd like**
Right click menu contain "Show In" option,  in "Show In"  have Project, Navigator sub options.  

**Describe alternatives you've considered**
manually search for files in the list.

**Additional context**
Please see standard Eclipse: 
![image](https://user-images.githubusercontent.com/899562/65978825-5ebe1200-e442-11e9-8833-1d12a611835e.png)

**Comment by uslss** [comment]
Link with editor option (Ctrl+Shift + ,) is already implemented for Database Navigator and Projects panel
